src/measure_theory/measure/measure_space_def.lean

```diff
@@ -360,6 +360,23 @@ lemma ae_le_set : s ≤ᵐ[μ] t ↔ μ (s \ t) = 0 :=
 calc s ≤ᵐ[μ] t ↔ ∀ᵐ x ∂μ, x ∈ s → x ∈ t : iff.rfl
            ... ↔ μ (s \ t) = 0          : by simp [ae_iff]; refl
 
+lemma ae_le_set_inter {s' t' : set α} (h : s ≤ᵐ[μ] t) (h' : s' ≤ᵐ[μ] t') :
+  (s ∩ s' : set α) ≤ᵐ[μ] (t ∩ t' : set α) :=
```

Comment:
This is true for any filter, so it's available as `h.inter h'`. Same for `ae_eq_inter`.

Reply:
Thanks a lot. I have kept the lemmas for ease of discoverability, but I can remove them if you prefer.